osm2streets/src/validate.rs

```diff
@@ -0,0 +1,52 @@
+use crate::StreetNetwork;
+
+impl StreetNetwork {
```

Comment:
Any other ideas for what we should check? Another idea is that `update_center_line`, `update_movements`, and (soon) `update_geometry` are all no-ops, to enforce that we're not forgetting to call something somewhere. (And I may open an issue about guaranteeing that in a more principled way)

Reply:
Not off the top of my head. But i love that there is a clear place to add any checks that come up while debugging.

Checking for no-op `update_` calls sounds very useful for enforcing those implicit dependencies, especially while trying to optimise where the update calls are made.